app/policies/application_policy.rb

```diff
@@ -15,11 +15,23 @@ def index?
   end
 
   def update?
-    @user.admin? ||  @record.user == @user
+    is_admin? ||  @record.user == @user
```

Comment:
Also, I notice that we have inconsistent instance var references in our policy files - that is, in some we refer to `@user` (like this one) and others we use the reader method `user` (like in `membership_application_policy.rb`).

We should be consistent in these references - I suggest the reader methods be used.